services/httpd/handler.go

```diff
@@ -204,6 +206,15 @@ func (h *Handler) Statistics(tags map[string]string) []models.Statistic {
 	}}
 }
 
+// TrackRequests begins tracking requests made to the Handler.
+// This should be called before this Handler is used to serve requests.
+func (h *Handler) TrackRequests() {
+	if h.requestTracker != nil {
+		return
+	}
+	h.requestTracker = NewRequestTracker()
```

Comment:
This can be removed. This was only used previously because the request tracker had a timer. Now it doesn't have a timer anymore so I can just move it to the handler creation.